scripts/variantstore/wdl/GvsCreateVATfromVDS.wdl

```diff
@@ -71,9 +65,15 @@ workflow GvsCreateVATfromVDS {
                 output_basename = vcf_filename
         }
 
+        call RemoveDuplicatesFromSitesOnlyVCF {
+            input:
+                sites_only_vcf = SelectVariants.output_vcf,
+                ref = reference
+        }
+
         call StripCustomAnnotationsFromSitesOnlyVCF {
             input:
-                input_vcf = SelectVariants.output_vcf,
+                input_vcf = RemoveDuplicatesFromSitesOnlyVCF.output_vcf,
```

Comment:
scatter looks good--I think the only thing I would like us to add is some additional validate vat queries because I think this opens us up to a few potential issues that we'll just want to track 
( based on sites or site-variants that were split between different shards or ended up that way because of left alignment )
1. Ensure that all the rows for a single site have the same AN value: Is there a site that has more than one distinct AN value? 
2. Ensure that all the rows for a specific site-variant / VID have the same AC value.  Is there a site-variant / VID that has more than one distinct AC value?